sensors/cron_sensor.yaml

```diff
@@ -0,0 +1,34 @@
+---
+  class_name: "CronSensor"
+  entry_point: "cron_sensor.py"
+  description: "Sensor to check to make sure our cron rules are running and succeeding"
+  poll_interval: 300
+  trigger_types:
+    -
+      name: "update_events"
```

Comment:
probably want to rename this to be more specific to this pack and what its doing.. maybe something like 'error_cron_event'